_data/tools.yml

```diff
@@ -23,6 +23,17 @@
   v2: true
   v3: true
 
+- name: unionj-generator
+  category: 
+    - dsl
+    - sdk
+    - code-generators
+  language: Java, Typescript...
+  description: OpenAPI 3.0 specification Dsl and a collection of code generators including spring boot controller/feign interface and typescript http sdk and more...
```

Comment:
Please correct 'Dsl' to 'DSL' and 'typescript' to 'TypeScript' and 'sdk to 'SDK'